tools/devops/automation/run-nightly-codeql.yml

```diff
@@ -0,0 +1,87 @@
+# xamarin-macios-nightly-codeql
+
+trigger: none
+pr: none
+
+schedules:
+- cron: "0 5 * * *"
+  displayName: Run daily at 5:00 UTC
+  branches:
+    include:
+    - main
+
+parameters:
+- name: macOSName  # comes from the build agent demand named macOS.Name
+  displayName: Name of the version of macOS to use
+  type: string
+  default: 'Sequoia'
+
+resources:
+  repositories:
+  - repository: yaml-templates
+    type: github
+    name: xamarin/yaml-templates
+    ref: refs/heads/main
+    endpoint: xamarin
+  - repository: maccore
+    type: github
+    name: xamarin/maccore
+    ref: refs/heads/main
+    endpoint: xamarin
```

Comment:
The maccore dependency will be removed soon (#22101), so maybe this PR can wait for that change (and then not add the maccore resource here)?

Reply:
Updated and removed this reference